apps/src/studio/collidable.js

```diff
@@ -104,6 +108,157 @@ Collidable.prototype.bounce = function () {
   }
 };
 
+function getRandomInt(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+Collidable.prototype.roamGrid = function(type) {
```

Comment:
This looks to be a fairly long function. It would be great to have a short comment explaining what it does, as well as what the expectations for the param "type" are.